public/scripts/group-chats.js

```diff
@@ -292,10 +292,11 @@ export function getGroupNames() {
 
 /**
  * Finds the character ID for a group member.
- * @param {string} arg 0-based member index or character name
- * @returns {number} 0-based character ID
+ * @param {number|string} arg 0-based member index or character name
+ * @param {Boolean} full Whether to return a key-value object containing extra data
+ * @returns {number|Object} 0-based character ID or key-value object if full is true
  */
-export function findGroupMemberId(arg) {
+export function findGroupMemberId(arg, full) {
```

Comment:
```suggestion
export function findGroupMemberId(arg, full = false) {
```
More clear on the definition that this is optional and defaults to false. Otherwise IDE is gonna complain on existing calls of that function.